### PR TITLE
Reinstate karma tests

### PR DESCRIPTION
**Pull request by rich-nguyen**
The tests were not running! As spotted by @Calanthe

### CODE REVIEW

File: static/test/javascripts/spec/common/weather.spec.js [hunk 1]
```diff
@@ -14,10 +14,10 @@ define([
     Injector
     ) {
 
-    return  new Injector()
+    return new Injector()
         .store('common/utils/config')
         .require(['common/modules/weather', 'mocks'], function (sut, mocks) {
-            ddescribe('Weather component', function () {
+            describe('Weather component', function () {
```
Comment (gklopper) [inline static/test/javascripts/spec/common/weather.spec.js]: Compiled languages #FTW

**Comment by gklopper** [comment]
:+1:

File: static/test/javascripts/spec/common/weather.spec.js (hunk 1 above)
Comment (ironsidevsquincy) [inline static/test/javascripts/spec/common/weather.spec.js]: this is actually valid in jasmine code; it means 'don't run this suite'. however, it seems to suppress this and all future ones - might be due to the way we're running them, need to investigate

Comment (rich-nguyen) [inline static/test/javascripts/spec/common/weather.spec.js]: Ah I see, although it looks like it means "only run suites prepended with d", which excluded the rest of our tests.

Comment (ironsidevsquincy) [inline static/test/javascripts/spec/common/weather.spec.js]: oh yes, you're right, not 'exclude' but 'only run'

**Pull request merged**